dev-packages/node-integration-tests/suites/express-v5/tracing/instrument-filterStatusCode.mjs

```diff
@@ -0,0 +1,14 @@
+import * as Sentry from '@sentry/node';
+import { loggingTransport } from '@sentry-internal/node-integration-tests';
+
+Sentry.init({
+  dsn: 'https://public@dsn.ingest.sentry.io/1337',
+  release: '1.0',
+  tracesSampleRate: 1.0,
+  transport: loggingTransport,
+  integrations: [
+    Sentry.httpIntegration({
+      dropSpansForIncomingRequestStatusCodes: [499, /3\d{2}/],
```

Comment:
The regex is an interesting API choice. I can kindof see the reason, just wanna suggest other options that might be a bit more intuitive to write and don't require googling what the regex for numbers is, like:

- Providing intervals to drop for ranges like `[from, to]` e.g. `[400, 499]`.
- Providing strings with `x` as a wildcard e.g. `"4xx"`

Reply:
yeah, I know what you mean. The drop ranges seem fine to me too, I'll adjust it to that instead!